pynest/examples/structural_plasticity.py

```diff
@@ -260,11 +260,10 @@ def record_ca(self):
 
 ####################################################################################
 # In order to save the state of the connectivity in the network through time
-# we create the function ``record_connectivity``. Here we use the ``GetStatus``
-# function to retrieve the number of connected pre-synaptic elements of each
-# neuron. The total amount of excitatory connections is equal to the total
-# amount of connected excitatory pre-synaptic elements. The same applies for
-# inhibitory connections.
+# we create the function ``record_connectivity``. Here we retrieve the number
+# of connected pre-synaptic elements of each neuron. The total amount of
+# excitatory connections is equal to the total amount of connected excitatory
+# pre-synaptic elements. The same applies for inhibitory connections.
 
     def record_connectivity(self):
         syn_elems_e = self.nodes_e.synaptic_elements
```

Comment:
Warning: "The PostScript backend does not support transparency; partially transparent artists will be rendered opaque."

Reply:
not nest-3 related feature, is going to be handled in a different PR, see #1504 checklist ... "check all plots".